lib/api/controllers/securityController.js

```diff
@@ -19,6 +19,68 @@ var
  */
 function SecurityController (kuzzle) {
   /**
+   * Get the role mapping
+   *
+   * @returns {Promise}
+   */
+  this.getRoleMapping = function securityGetRoleMapping () {
+    return kuzzle.internalEngine.getMapping({index: kuzzle.internalEngine.index, type: 'roles'})
+      .then(response => ({mapping: response[kuzzle.internalEngine.index].mappings.roles.properties}));
+  };
+
+  /**
+   * Update the roles collection mapping
+
+   * @param {Request} request
+   * @returns {Promise}
+   */
+  this.updateRoleMapping = function securityUpdateRoleMapping (request) {
+    return kuzzle.internalEngine.updateMapping('roles', request.input.body);
+  };
+
+  /**
+   * Get the profile mapping
+   *
+   * @returns {Promise}
+   */
+  this.getProfileMapping = function securityGetProfileMapping () {
+    return kuzzle.internalEngine.getMapping({index: kuzzle.internalEngine.index, type: 'profiles'})
+      .then(response => {
+        return Promise.resolve({mapping: response[kuzzle.internalEngine.index].mappings.profiles.properties});
```

Comment:
Why not just `return {mapping: response[kuzzle.internalEngine.index].mappings.profiles.properties};`?